contributing/workflow.qmd

```diff
@@ -622,39 +646,38 @@ Using `.qmd` there are also ways to control execution cell-by-cell via
 
 ### Quarto (non)executable syntax
 
-Code chunks are executed if they are within curly brackets like `{python}`. If we don't want code in a `.qmd` or `.Rmd` file to be executed, we use the syntax:
+Code chunks are executed if they are within curly brackets like
+`{python}`. If we don't want code in a `.qmd` or `.Rmd` file to be
+executed, we use the syntax:
 
-```python
-```python
-```
-
-This is required even if the YAML at the top of the file has `execute: false`.
+![](images/backtick-syntax.png){width="15%"}
```

Comment:
Hey @jules32 there's a trick for displaying code fences. Wrap them in another, larger code fence!

`````````
``````
```python
print("Hello")
```
``````
`````````

Reply:
@mfisher87 Thank you I never thought of exaggerated fences! 
We used an image this time cause one instance of fences to display ` ```python` resulted in altered txt colour below it. I'll try when I revisit this

Reply:
Looking at [the lines which originally caused the shading problem](https://github.com/NASA-Openscapes/earthdata-cloud-cookbook/pull/257/commits/3f03fbe6acb24630e2b2952b39ed6f4a98da7c59#diff-4e956f2bfa8ad914b24b6b2994a9a2afd9ed70432ef94c60b2aa0ac0f25fc249L627-L629), I _think_ it's because the outer code fences specify python syntax:

``````
```python
```python
```
``````


If the outer fences are larger than the inner fences _and_ don't have a syntax indicator, like this, I think (hope :laughing:) the shading problem wouldn't occur here. Of course this could be caused by a bug in Quarto, in which case my suggestion may not help :P

`````````
``````
```python
``````
`````````